pkg/apis/infinispan/v1/types_util.go

```diff
@@ -32,6 +33,8 @@ const (
 	PodTargetLabels string = "infinispan.org/podTargetLabels"
 	// TargetLabels labels propagated to services/ingresses/routes
 	TargetLabels string = "infinispan.org/targetLabels"
+	// ServiceMonitoringLabel defines if we need to create ServiceMonitor or not
+	ServiceMonitoringLabel string = "infinispan.org/monitoring"
```

Comment:
@dmvolod For end users would this work by setting this in the Infinispan CR like:

```
metadata:
  annotations:
    infinispan.org/monitoring: true
```

Reply:
Yes, @oraNod , will look like, `true` is string
```
metadata:
  annotations:
    infinispan.org/monitoring: 'true'
```